smoketest.bash

```diff
@@ -36,7 +35,7 @@ display_usage() {
     echo -e "\t-s [seaweed|minio|cloudserver|localstack]\tS3 implementation to spin up (default \"seaweed\")."
     echo -e "\t-G\t\t\t\t\t\texclude Grafana dashboard and jfr-datasource from deployment."
     echo -e "\t-r\t\t\t\t\t\tconfigure a cryostat-Reports sidecar instance"
-    echo -e "\t-t\t\t\t\t\t\tinclude sample applications for Testing."
+    echo -e "\t-t [all|comma-list]\t\t\t\tinclude sample applications for Testing. Leave blank or use 'all' to deploy everything, otherwise use a comma-separated list from:\n\t\t\t\t\t\t\t\t$(find "${DIR}/compose/sample_apps" -type f -name '*.yml' -exec basename {} \; | cut -d. -f1 | grep -v https | sort | tr '\n' ',' | sed 's/,$//')"
```

Comment:
While you are it, maybe format the help message to all start with capitalized word?

Reply:
I started with them all lowercase on purpose actually, with words capitalized as a mnemonic device: `-t` for **T**esting, `-s` for **S**3, `-O` for **O**ffline, etc. Whether the flag itself is uppercase or lowercase is a bit arbitrary. The only exception to this rule right now is `-k`, which is like "inse**k**ure". I asked Ming to use `-k` for this because it's similar in meaning to `curl -k`

Reply:
Nicee! That works better!